sklearn/ensemble/forest.py

```diff
@@ -1631,6 +1631,28 @@ class labels (multi-output problem).
     reduce memory consumption, the complexity and size of the trees should be
     controlled by setting those parameter values.
 
+    Examples
+    --------
+    >>> from sklearn.linear_model import ExtraTreesClassifier
+    >>> from sklearn.datasets import make_classification
+
+    >>> X, y = make_classification(n_features=4, random_state=0)
+    >>> clf = ExtraTreesClassifier(n_estimators = 100, 
+                                    random_state = 100)
+    >>> clf.fit(X, y)  # doctest: +NORMALIZE_WHITESPACE
+    ExtraTreesClassifier(bootstrap=False, class_weight=None, criterion='gini',
+           max_depth=None, max_features='auto', max_leaf_nodes=None,
+           min_impurity_decrease=0.0, min_impurity_split=None,
+           min_samples_leaf=1, min_samples_split=2,
+           min_weight_fraction_leaf=0.0, n_estimators=100, n_jobs=None,
+           oob_score=False, random_state=100, verbose=0, warm_start=False)
+           
+    >>> print(clf.coef_)
```

Comment:
ExtraTreesClassifier does not have a coef_ attribute. Furthermore I read that print is not necessary for printing results in the REPL